bin/internal/flutter.version

```diff
@@ -1 +1 @@
-cf97c772eecef5cdc388f5457631fb4ae071e68d
+babf4897cd746f169ebc7c079c8af4515d726a92
```

Comment:
Is this correct? Looking at https://github.com/shorebirdtech/flutter/tree/flutter_release/3.10.4, HEAD is `cf97c772eecef5cdc388f5457631fb4ae071e68d`